test/pirls.jl

```diff
@@ -46,12 +39,12 @@ end
 @testset "cbpp" begin
     cbpp = dat[:cbpp]
     cbpp[!, :prop] = cbpp[!, :i] ./ cbpp[!, :s]
-    gm2 = fit(MixedModel, @formula(prop ~ 1 + p + (1|h)), cbpp, Binomial(), wts = cbpp[!,:s])
-    @test isapprox(deviance(gm2,true), 100.09585619324639, atol=0.0001)
-    @test isapprox(sum(abs2, gm2.u[1]), 9.723175126731014, atol=0.0001)
-    @test isapprox(logdet(gm2), 16.90099, atol=0.0001)
-    @test isapprox(sum(gm2.resp.devresid), 73.47179193718736, atol=0.001)
-    @test isapprox(loglikelihood(gm2), -92.02628186555876, atol=0.001)
+    gm2 = fit(MixedModel, @formula(prop ~ 1 + p + (1|h)), cbpp, Binomial(), wts=cbpp[!,:s])
+    @test isapprox(deviance(gm2,true), 100.09585619892968, atol=0.0001)
+    @test isapprox(sum(abs2, gm2.u[1]), 9.723054788538546, atol=0.0001)
+    @test isapprox(logdet(gm2), 16.90105378801136, atol=0.0001)
+    @test isapprox(sum(gm2.resp.devresid), 73.47174762237978, atol=0.001)
+    @test isapprox(loglikelihood(gm2), -92.02628186840045, atol=0.001)
```

Comment:
Why did the reference values change?

Reply:
Slightly different paths for the optimization because of different starting estimates.  It seems that the deviance at the optimum is reasonably well determined but the individual components can trade off against one another to some extent.

Reply:
Right, that might explain the difficulties I'm having with the tests I'm working on -- the lme4 estimates have deviance close to those from this branch, but the estimates vary a fair amount. And it might also explain why the old gamma tests in #201 gave different answers on different platforms.